From 7a66629e44973c2e0de3ee0d902100a557455ee9 Mon Sep 17 00:00:00 2001
From: Daniele Tricoli <eriol@mornie.org>
Date: Thu, 8 Oct 2015 13:19:46 -0700
Subject: Do not use embedded copy of python-six.

Forwarded: not-needed

Patch-Name: 01_do-not-use-embedded-python-six.patch
---
 dummyserver/handlers.py                        | 2 +-
 test/__init__.py                               | 2 +-
 test/contrib/test_pyopenssl.py                 | 2 +-
 test/test_collections.py                       | 2 +-
 test/test_fields.py                            | 2 +-
 test/test_filepost.py                          | 2 +-
 test/test_retry.py                             | 2 +-
 test/with_dummyserver/test_chunked_transfer.py | 2 +-
 test/with_dummyserver/test_connectionpool.py   | 2 +-
 test/with_dummyserver/test_https.py            | 2 +-
 urllib3/_collections.py                        | 2 +-
 urllib3/connection.py                          | 2 +-
 urllib3/connectionpool.py                      | 2 +-
 urllib3/contrib/appengine.py                   | 2 +-
 urllib3/fields.py                              | 2 +-
 urllib3/filepost.py                            | 4 ++--
 urllib3/response.py                            | 4 ++--
 urllib3/util/request.py                        | 2 +-
 urllib3/util/response.py                       | 2 +-
 urllib3/util/retry.py                          | 2 +-
 20 files changed, 22 insertions(+), 22 deletions(-)

diff --git a/dummyserver/handlers.py b/dummyserver/handlers.py
index 7598329..504b998 100644
--- a/dummyserver/handlers.py
+++ b/dummyserver/handlers.py
@@ -269,7 +269,7 @@ def _parse_header(line):
     """
     import tornado.httputil
     import email.utils
-    from urllib3.packages import six
+    import six
     if not six.PY3:
         line = line.encode('utf-8')
     parts = tornado.httputil._parseparam(';' + line)
diff --git a/test/__init__.py b/test/__init__.py
index bab39ed..076cdf0 100644
--- a/test/__init__.py
+++ b/test/__init__.py
@@ -8,7 +8,7 @@ import socket
 from nose.plugins.skip import SkipTest
 
 from urllib3.exceptions import MaxRetryError, HTTPWarning
-from urllib3.packages import six
+import six
 
 # We need a host that will not immediately close the connection with a TCP
 # Reset. SO suggests this hostname
diff --git a/test/contrib/test_pyopenssl.py b/test/contrib/test_pyopenssl.py
index ab304f8..b9e6572 100644
--- a/test/contrib/test_pyopenssl.py
+++ b/test/contrib/test_pyopenssl.py
@@ -1,5 +1,5 @@
 from nose.plugins.skip import SkipTest
-from urllib3.packages import six
+import six
 
 try:
     from urllib3.contrib.pyopenssl import (inject_into_urllib3,
diff --git a/test/test_collections.py b/test/test_collections.py
index 9d72939..78ef634 100644
--- a/test/test_collections.py
+++ b/test/test_collections.py
@@ -4,7 +4,7 @@ from urllib3._collections import (
     HTTPHeaderDict,
     RecentlyUsedContainer as Container
 )
-from urllib3.packages import six
+import six
 xrange = six.moves.xrange
 
 from nose.plugins.skip import SkipTest
diff --git a/test/test_fields.py b/test/test_fields.py
index 21b4481..b547aa8 100644
--- a/test/test_fields.py
+++ b/test/test_fields.py
@@ -1,7 +1,7 @@
 import unittest
 
 from urllib3.fields import guess_content_type, RequestField
-from urllib3.packages.six import u, PY3
+from six import u, PY3
 from . import onlyPy2
 
 
diff --git a/test/test_filepost.py b/test/test_filepost.py
index 390dbb3..ecc6710 100644
--- a/test/test_filepost.py
+++ b/test/test_filepost.py
@@ -2,7 +2,7 @@ import unittest
 
 from urllib3.filepost import encode_multipart_formdata, iter_fields
 from urllib3.fields import RequestField
-from urllib3.packages.six import b, u
+from six import b, u
 
 
 BOUNDARY = '!! test boundary !!'
diff --git a/test/test_retry.py b/test/test_retry.py
index 1e87585..48bad37 100644
--- a/test/test_retry.py
+++ b/test/test_retry.py
@@ -1,7 +1,7 @@
 import unittest
 
 from urllib3.response import HTTPResponse
-from urllib3.packages.six.moves import xrange
+from six.moves import xrange
 from urllib3.util.retry import Retry
 from urllib3.exceptions import (
     ConnectTimeoutError,
diff --git a/test/with_dummyserver/test_chunked_transfer.py b/test/with_dummyserver/test_chunked_transfer.py
index 1d58e23..abfc777 100644
--- a/test/with_dummyserver/test_chunked_transfer.py
+++ b/test/with_dummyserver/test_chunked_transfer.py
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 
 from urllib3 import HTTPConnectionPool
-from urllib3.packages import six
+import six
 from dummyserver.testcase import SocketDummyServerTestCase
 
 
diff --git a/test/with_dummyserver/test_connectionpool.py b/test/with_dummyserver/test_connectionpool.py
index 0f31fa0..b3b72ca 100644
--- a/test/with_dummyserver/test_connectionpool.py
+++ b/test/with_dummyserver/test_connectionpool.py
@@ -31,7 +31,7 @@ from urllib3.exceptions import (
     ProtocolError,
     NewConnectionError,
 )
-from urllib3.packages.six import b, u
+from six import b, u
 from urllib3.util.retry import Retry
 from urllib3.util.timeout import Timeout
 
diff --git a/test/with_dummyserver/test_https.py b/test/with_dummyserver/test_https.py
index 2c5f035..be54e3c 100644
--- a/test/with_dummyserver/test_https.py
+++ b/test/with_dummyserver/test_https.py
@@ -36,7 +36,7 @@ from urllib3.exceptions import (
     SystemTimeWarning,
     InsecurePlatformWarning,
 )
-from urllib3.packages import six
+import six
 from urllib3.util.timeout import Timeout
 import urllib3.util as util
 
diff --git a/urllib3/_collections.py b/urllib3/_collections.py
index 77cee01..114116c 100644
--- a/urllib3/_collections.py
+++ b/urllib3/_collections.py
@@ -15,7 +15,7 @@ try:  # Python 2.7+
     from collections import OrderedDict
 except ImportError:
     from .packages.ordered_dict import OrderedDict
-from .packages.six import iterkeys, itervalues, PY3
+from six import iterkeys, itervalues, PY3
 
 
 __all__ = ['RecentlyUsedContainer', 'HTTPHeaderDict']
diff --git a/urllib3/connection.py b/urllib3/connection.py
index 5ce0080..f4b86fa 100644
--- a/urllib3/connection.py
+++ b/urllib3/connection.py
@@ -6,7 +6,7 @@ import sys
 import socket
 from socket import error as SocketError, timeout as SocketTimeout
 import warnings
-from .packages import six
+import six
 
 try:  # Python 3
     from http.client import HTTPConnection as _HTTPConnection
diff --git a/urllib3/connectionpool.py b/urllib3/connectionpool.py
index ab634cb..652b297 100644
--- a/urllib3/connectionpool.py
+++ b/urllib3/connectionpool.py
@@ -31,7 +31,7 @@ from .exceptions import (
     NewConnectionError,
 )
 from .packages.ssl_match_hostname import CertificateError
-from .packages import six
+import six
 from .connection import (
     port_by_scheme,
     DummyConnection,
diff --git a/urllib3/contrib/appengine.py b/urllib3/contrib/appengine.py
index 1579476..22e76e0 100644
--- a/urllib3/contrib/appengine.py
+++ b/urllib3/contrib/appengine.py
@@ -12,7 +12,7 @@ from ..exceptions import (
     SSLError
 )
 
-from ..packages.six import BytesIO
+from six import BytesIO
 from ..request import RequestMethods
 from ..response import HTTPResponse
 from ..util.timeout import Timeout
diff --git a/urllib3/fields.py b/urllib3/fields.py
index 8fa2a12..14e5604 100644
--- a/urllib3/fields.py
+++ b/urllib3/fields.py
@@ -2,7 +2,7 @@ from __future__ import absolute_import
 import email.utils
 import mimetypes
 
-from .packages import six
+import six
 
 
 def guess_content_type(filename, default='application/octet-stream'):
diff --git a/urllib3/filepost.py b/urllib3/filepost.py
index 97a2843..2fea190 100644
--- a/urllib3/filepost.py
+++ b/urllib3/filepost.py
@@ -4,8 +4,8 @@ import codecs
 from uuid import uuid4
 from io import BytesIO
 
-from .packages import six
-from .packages.six import b
+import six
+from six import b
 from .fields import RequestField
 
 writer = codecs.lookup('utf-8')[3]
diff --git a/urllib3/response.py b/urllib3/response.py
index 5567903..10e14ec 100644
--- a/urllib3/response.py
+++ b/urllib3/response.py
@@ -9,8 +9,8 @@ from ._collections import HTTPHeaderDict
 from .exceptions import (
     ProtocolError, DecodeError, ReadTimeoutError, ResponseNotChunked
 )
-from .packages.six import string_types as basestring, binary_type, PY3
-from .packages.six.moves import http_client as httplib
+from six import string_types as basestring, binary_type, PY3
+from six.moves import http_client as httplib
 from .connection import HTTPException, BaseSSLError
 from .util.response import is_fp_closed, is_response_to_head
 
diff --git a/urllib3/util/request.py b/urllib3/util/request.py
index 7377931..40bf0b4 100644
--- a/urllib3/util/request.py
+++ b/urllib3/util/request.py
@@ -1,7 +1,7 @@
 from __future__ import absolute_import
 from base64 import b64encode
 
-from ..packages.six import b
+from six import b
 
 ACCEPT_ENCODING = 'gzip,deflate'
 
diff --git a/urllib3/util/response.py b/urllib3/util/response.py
index 0b5c75c..d7759ae 100644
--- a/urllib3/util/response.py
+++ b/urllib3/util/response.py
@@ -1,5 +1,5 @@
 from __future__ import absolute_import
-from ..packages.six.moves import http_client as httplib
+from six.moves import http_client as httplib
 
 from ..exceptions import HeaderParsingError
 
diff --git a/urllib3/util/retry.py b/urllib3/util/retry.py
index d379833..c2e5380 100644
--- a/urllib3/util/retry.py
+++ b/urllib3/util/retry.py
@@ -9,7 +9,7 @@ from ..exceptions import (
     ReadTimeoutError,
     ResponseError,
 )
-from ..packages import six
+import six
 
 
 log = logging.getLogger(__name__)
